conf/nginx.conf

```diff
@@ -69,15 +69,15 @@ location ^~ __PATH__/ {
     access_log off;
   }
 
-  location ~ ^__PATH__/(?:build|tests|config|lib|3rdparty|templates|data)/ {
+  location ~ ^\__PATH__\/(?:build|tests|config|lib|3rdparty|templates|data)\/ {
```

Comment:
I'm randomly jumping in but won't this trigger weird issues considering that the nginx helper replaces `__PATH__` and `__PATH__/` by different thing in the case of installation on a domain root ? Or is it in fact on purpose ?

Reply:
but kay0u it was tested... was it? :sweat_smile:

Reply:
https://github.com/YunoHost/yunohost/blob/051d14a945ef00317cf79061389fb15b0fee951b/data/helpers.d/nginx#L35-L36

What kind of different thing? I tested it, it works, but I have to admit that I just copy pasta the current example, and I'm not sure if we should keep these `\`